index.js

```diff
@@ -562,6 +562,7 @@ class Plugin extends MiniAccountsPlugin {
         amount: OUTGOING_CHANNEL_DEFAULT_AMOUNT
       })
         .then(async () => {
+          account.setClientChannel(clientChannel) // reloads the channel amount
```

Comment:
This is an async call; shouldn't it be `await`ed here?

Reply:
Fixed